shopinvader_sale_coupon/tests/test_cart.py

```diff
@@ -196,27 +313,23 @@ def test_promotion_on_next_order(self):
             program.id,
             "Coupons for next order should've been generated",
         )
-        self.assertEqual(
-            generated_coupon["partner"]["id"],
-            self.partner.id,
-            "Coupons for next order should've been generated for this partner",
```

Comment:
Now coupons are not associed to a partner anymore. See https://github.com/odoo/odoo/commit/6927dcda9f99db2a4d2f06b32f8d377c17fe2794